tests/direct_adaptor_factory_wrapper.py

```diff
@@ -0,0 +1,133 @@
+import json
+import logging
+import time
+import sys
+import utils.utility as enclave_helper
+import utils.signature as signature
+from service_client.generic import GenericServiceClient
+import workorder_json
+from tcf_connector.direct_json_rpc_api_adaptor_factory import DirectJsonRpcApiAdaptorFactory
+import utils.hex_utils as hex_utils
+
+# Remove duplicate loggers
+for handler in logging.root.handlers[:]:
+    logging.root.removeHandler(handler)
+logger = logging.getLogger(__name__)
+# -----------------------------------------------------------------
+class DirectAdaptorFactoryWrapper:
+	# Contains the functions needed to submit and retrieve work orders in the Direct Model
+	def __init__(self, config_file):
+		self.adaptor_factory = DirectJsonRpcApiAdaptorFactory(config_file)
+		self.worker_registry_list = None
+		self.worker_registry = None
+		self.work_order = None
+		self.work_order_receipt = None
+
+	def init_worker_registry_list(self, config):
```

Comment:
Can't we initialize this in constructor?

Reply:
No, because the config needs to be updated with the retrieved worker registry URI from the worker registry list adaptor before initializing the worker registry adaptor. See common/tcf_connector/direct_json_rpc_api_adaptor_factory.py: all adaptors start as None and cannot be updated after being initialized.